guessingGame.js

```diff
@@ -0,0 +1,20 @@
+function GuessingGame(num) {
+  this.answer = num;
+  this.isSolved = function() {
```

Comment:
I would make a variable `this.solved = false` then just set `this.solved = true` when the guess is correct then have `this.isSolved = function() { return this.solved}`